vmpooler.yaml.example

```diff
@@ -176,6 +179,7 @@
     - 'created_by'
     - 'project'
   domain: 'company.com'
+  prefix: ''
```

Comment:
The value in the example configuration file should be populated with an example value, similar to `domain` on the line above. Please include an example prefix as this setting is intended to be used.